src/openforms/formio/rendering/default.py

```diff
@@ -311,7 +311,6 @@ def get_children(self) -> Iterator["ComponentNode"]:
 
     def __post_init__(self):
         self._base_label = self.component.get("groupLabel", self.default_label)
-        super().__post_init__()
```

Comment:
Yes,I removed the `__post_init__` from the `ComponentNode` so that's why it was removed here as well.
[https://github.com/open-formulieren/open-forms/pull/3960/files/24efb00ce3b30699dc4529c23e14eb34f5ed673e#diff-5fa3e1ab66d508ceaa4cb2a42ce27a9339cf2f4ad82e038fd6fb4272c52e318fL87](url)